tests/unit/ssg-module/test_common.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+import ssg.entities.common as common
+
+
+def test_make_items_product_specific():
+    qualified_dict = {
+        "name@foo": "Name: Foo",
+        "title@bar": "Title: Bar",
+        "title@foo": "Title: Foo",
+        "meta": "Common meta",
+    }
+    qualified_dict.update({
```

Comment:
Anybody looking at the test later won't understand what is that supposed to mean. It is probably needed for tests to pass, but shouldn't the code be fixed not to be sensitive to ordering issues while we are touching it anyway?

Reply:
Explanation added.